### PR TITLE
Update pyface and traitsui version requirement following last release.

### PR DESCRIPTION
**Pull request by ericpre**
pyface and traitsui version 6.0.0 are out!

### CODE REVIEW

**Comment by jat255** [comment]
Woo! +1 +1 +1

File: README.rst [hunk 1]
```diff
@@ -46,8 +46,8 @@ and its dependencies, although it might not support older versions.
 
 Installation via pip and conda
 ------------------------------
-If you want to install HyperSpyUI with pyqt5, you will need to install the 
-lastest version or pyface (>= 6.0.0) and traitsui (>= 5.2.0).
+
+HyperSpyUI requires pyqt4 or pyqt5.
```
[the remark this answers is not in the document]
Comment (ericpre) [reply README.rst]: What would be the version name for the next release? If we know it, I could set the correct version name here already.
Out of curiosity, I have tried to install HyperSpyUI from an official python distribution on windows and the two libraries that I could not install directly from pip are traits and statsmodels. Considering that statsmodels could be set as an optional dependencies of hyperspy, it is not as bad as I would have expected! ;)

[the remark this answers is not in the document]
Comment (francisco-dlp) [reply README.rst]: @vidartf, would you advice to hold the release of the bundle until the threaded progress bar bugs gets fixed?

**Comment by ericpre** [comment]
I have tried to reproduce the seg fault on windows by running a decomposition and I can't reproduce!

**Comment by jat255** [comment]
I would advise a new release ASAP, since the current installation instructions don't work as written.

**Comment by ericpre** [comment]
I have tried to reproduce the seg fault by different means and I wasn't successful... Could someone provide an example? In any case, a new release would be very welcome to avoid having to downgrade pyqt5 (this branch is used to build the bundle).

**Comment by jat255** [comment]
@ericpre so I just tried to reproduce this on my Windows system with the same datafile, and everything worked fine. I'll explore more when I get back to my Linux machine to see if I can figure out what's going on.

**Comment by jat255** [comment]
Ok, I just confirmed this on my Linux VM. I installed from scratch with the following:

```bash
conda create -n hyperspy python=3.6
activate hyperspy
conda install hyperspy -c conda-forge  # (to get all the dependencies, then I install development version with pip)
conda remove hyperspy
conda install traitsui pyface
pip install -e ./hyperspy     #(@RELEASE_next_minor)
pip install -e --no-deps ./hyperspy_gui_ipywidgets     #(@master)
pip install -e --no-deps ./hyperspy_gui_traitsui     #(@master)
pip install -e ./hyperspyUI    #(@master)
```

I then load the SEM-EDS map, rebin using Signal -> Rebin (4x4x2), click BSS, then click convert, and get a segfault. Any ideas on how to debug?

**Comment by jat255** [comment]
`QXcbConnection: XCB error: 3 (BadWindow), sequence: 31117, resource id: 33564893, major code: 40 (TranslateCoords), minor code: 0`

This was in the console around the same time as the segfault.

**Comment by vidartf** [comment]
I'm pretty sure we're doing something wrong when updating the progress bar from a worker thread. There's quite a few things that need to be taken into account when communicating across threads, and you need to be careful about which threads own which Qt objects. It's always been a little tricky.

**Comment by jat255** [comment]
Ok, I made an issue specifically about this in #161

File: README.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (vidartf) [reply README.rst]: TBH, the specific version number doesn't matter that much, as currently the plugin system doesn't allow a way of specifying version constraints (and plugins should be the only thing that uses HUI as a library/API). 1.1 or 2.0 are both acceptable, but maybe 1.1 makes more sense from a user perspective.

[the remark this answers is not in the document]
Comment (ericpre) [reply README.rst]: 1.1 would make more sense to me, since this is mainly a maintenance release.

**Comment by ericpre** [comment]
@vidartf: now that we have fixed a few critical issue, would you like to make a new release so we can remove get ride of the confusing installation issue.
I may try to have a look at the thread issue in the next couple of weeks.

**Comment by vidartf** [comment]
@ericpre I want to test-drive it a little first, but yes, that sounds reasonable. I also want to have a look at the update checker code, as that will likely be broken now with pip 10+.

**Comment by vidartf** [comment]
PS: For the update checker code, if the solution is non-trivial I will simply remove/disable the relevant code.

**Comment by ericpre** [comment]
Done in #174 and #177.

**Pull request closed without merging**